src/common/k8s-api/create-json-api.injectable.ts

```diff
@@ -18,8 +20,21 @@ const createJsonApiInjectable = getInjectable({
       fetch: di.inject(fetchInjectable),
       logger: di.inject(loggerInjectable),
     };
+    const lensProxyCert = di.inject(lensProxyCertificateInjectionToken);
 
-    return (config, reqInit) => new JsonApi(deps, config, reqInit);
+    return (config, reqInit) => {
+      config.getRequestOptions = async () => {
```

Comment:
We shouldn't be overriding something that might be set

Reply:
Good catch, will fix.

Reply:
Fixed.